netmanager/src/views/pages/Map/OverlayMap.js

```diff
@@ -388,15 +385,20 @@ export const OverlayMap = ({ center, zoom, heatMapData, monitoringSiteData }) =>
   const [showSensors, setShowSensors] = useState(true);
   const [showCalibratedValues, setShowCalibratedValues] = useState(false);
   const [showHeatMap, setShowHeatMap] = useState(false);
+  const mapContainerRef = useRef(null);
   const initialShowPollutant = {
-    pm2_5: handleLocalStorage.get('pollutant') === 'pm2_5',
-    no2: handleLocalStorage.get('pollutant') === 'no2',
-    pm10: handleLocalStorage.get('pollutant') === 'pm10'
+    pm2_5: false,
```

Comment:
@OchiengPaul442 , also for this section, what is happening when these pollutants are assigned to false?